Dockerfile

```diff
@@ -1,33 +1,55 @@
-FROM ubuntu:21.04
+FROM ubuntu:20.04
```

Comment:
Curious about the move from 21 to 20 here.  Due to the upcoming EOL of 20.04 would it make sense to work on bumping this to 22.04 or higher?

Reply:
Version of ubuntu in Docker `webodm_db` and `webodm_webapp` should be same to ensure apt will install some package in the same version. Maybe I should to update both?

Reply:
Done. The base image is Ubuntu 22.04 now, because PDAL can not be found by apt in Ubuntu 24.04.

Reply:
Yeah 24 is still fairly new but 22 has plenty of life left in it.